include/boost/spirit/home/x3/char/char_class.hpp

```diff
@@ -71,7 +71,7 @@ namespace boost { namespace spirit { namespace x3
 
 #define BOOST_SPIRIT_X3_CHAR_CLASS(encoding, name)                                 \
     typedef char_class<char_encoding::encoding, name##_tag> name##_type;           \
-    name##_type const name = name##_type();                                        \
+    constexpr name##_type name = name##_type();                                        \
```

Comment:
Since you have denied edits to your branch, please fix backslashes indentation.

Reply:
fixed in new commit [ad88408](https://github.com/boostorg/spirit/pull/570/commits/ad88408b4571ffb0144589c2cefa3f6ae68f4144)